docs/overview.md

```diff
@@ -194,6 +196,8 @@ Using a variant                 | `let x = Two("Hello", "World");`
 
 ## Options
 
+_Details: [Options](option.md)_
```

Comment:
I like this format of a brief overview page with links to details. (Kind of wish our docs could expand the sections inline on demand) (or, the entire doc could compress into the equivalent of the overview page at will / by default).